index.js

```diff
@@ -57,3 +63,17 @@ function tryParseFloat(value) {
 	}
 	return result;
 }
+
+function getTypeFromUnit ( unit ) {
+	if ( cssAngleUnits.indexOf(unit) !== -1 ) {
+		return 'angle';
+	} else if ( cssResolutionUnits.indexOf(unit) !== -1 ) {
```

Comment:
That's definitely concise; though, it's not as performant, because you aren't using a hash lookup. The main thing I'm trying to avoid in my example is `units.indexOf(unit) !== -1`, which has to be done a number of times until it finds the right one. If you used a lookup table, it would just search for the unit as key to that table and be done w/ it.